lib/src/base-styles/_nx-globals.scss

```diff
@@ -18,19 +18,45 @@ $-nx-scrollable-default-height: 400px;
   color: var(--nx-color-link);
   cursor: pointer;
 
+  .nx-code {
+    color: var(--nx-color-link);
+  }
+
   &:hover {
     color: var(--nx-color-link-hover);
+
+    .nx-code {
+      color: var(--nx-color-link-hover);
+    }
   }
 
   &:active {
     color: var(--nx-color-link-active);
+
+    .nx-code {
+      color: var(--nx-color-link-active);
+    }
   }
 
   &:focus {
-    border-radius: var(--nx-border-radius);
+    // outline-offset scales the border-radius so this
+    // was changed to 1px to get it visually close to
+    // var(--nx-border-radius) or (6px).
+    border-radius: 1px;
     color: var(--nx-color-link-focus);
     outline: 1px solid var(--nx-color-interactive-border-focus);
-    outline-offset: 2px;
+    outline-offset: 4px;
+
+    .nx-code {
+      color: var(--nx-color-link-focus);
+    }
+  }
+
+  .nx-icon {
+    // Offsets the icon down to make it
+    // more optically aligned vertically
+    // to the middle.
+    margin-bottom: -1px;
```

Comment:
FA icons have a custom vertical-align value defined in ems.  I think that any custom adjustment of vertical positioning of FA icons that we do should also be done with vertical-align to keep things simple.  I also think it should be done in ems so that it works at different font sizes.

Reply:
> FA icons have a custom vertical-align value defined in ems. I think that any custom adjustment of vertical positioning of FA icons that we do should also be done with vertical-align to keep things simple. I also think it should be done in ems so that it works at different font sizes.

Updated to use vertical-align and em here: Should this apply to any .nx-icon within a link or just the external link icon?